svcomp-benchmark/README.md

```diff
@@ -1,8 +1,9 @@
-The [SV-benchmarks](https://github.com/sosy-lab/sv-benchmarks/tree/master/c) consist of a large number of C programs that are used as verification tasks during the [SV-COMP](http://sv-comp.sosy-lab.org/2016/benchmarks.php). We analyzed 1346 programs classified as correct from the benchmarks, and ran them after compiling with [RV-Match](http://runtimeverification.com/match). For more information on how the programs were chosen, please see the [results](results/) directory. For information on how we compiled and ran the programs with RV-Match, please see the [scripts](scripts/) directory. 
+The [SV-benchmarks](https://github.com/sosy-lab/sv-benchmarks/tree/master/c) consist of a large number of C programs that are used as verification tasks during the [SV-COMP](http://sv-comp.sosy-lab.org/2016/benchmarks.php). We analyzed 1346 programs classified as correct from the benchmarks, and ran them after compiling with [RV-Match](http://runtimeverification.com/match). For more information on how the programs were chosen, please see the [results](results/) directory. For information on how we compiled and ran the programs with RV-Match, please see the [scripts](scripts/) directory. If you'd like to run RV-Match on the benchmarks yourself, or use RV-Match to run more experiments, then you can obtain the latest version of the tool (Academic/Evaluation licenses available) from [RV Inc.'s website] (https://runtimeverification.com/match/download/).
```

Comment:
[RV Inc.'s website] (https://runtimeverification.com/match/download/).

Is the link still going to work with a space there?